package.json

```diff
@@ -6,13 +6,14 @@
   "main": "src/main.js",
   "homepage": "./",
   "scripts": {
-    "start": "npx electron . --dev",
```

Comment:
Was the reason for dropping `npx` just because it's not really needed?

Reply:
Yeah, `npx` was pretty redundant inside this "scripts" context. I mainly use `npx` because it searches the local node_modules first for CLIs like electron, etc. But the "scripts" context already ensures that, I've since learned.